templates/users/default.dev.j2

```diff
@@ -2,13 +2,15 @@
 
 {%- set config = item.config|default([]) -%}
 
+# If you come from bash you might have to change your $PATH.
+{{ macros.config_row(config, 'export PATH', '# export PATH=$HOME/bin:/usr/local/bin:$PATH', true) }}
+
 # Path to your oh-my-zsh installation.
 {{ macros.config_row(config, 'export ZSH', manala_ohmyzsh_dir) }}
 
-# Set name of the theme to load.
-# Look in ~/.oh-my-zsh/themes/
-# Optionally, if you set this to "random", it'll load a random theme each
-# time that oh-my-zsh is loaded.
+# Set name of the theme to load. Optionally, if you set this to "random"
+# it'll load a random theme each time that oh-my-zsh is loaded.
```

Comment:
From [oh-my-zsh/wiki/Customization#overriding-and-adding-themes](https://github.com/robbyrussell/oh-my-zsh/wiki/Customization#overriding-and-adding-themes):

> Note: Using a random theme with $ZSH_THEME="random" will not look into your custom themes directory. Only built-in themes will be used. (Until PR #3743 is merged, which makes random include custom themes.)

It's the same behavior here, right? If it is, I think it may be good to mention, or at least change the link just below to this one which looks more adapted imho.